backend/src/near.js

```diff
@@ -25,12 +25,16 @@ const queryFinalBlock = async () => {
 };
 
 const queryEpochStats = async () => {
+  const networkGenesisConfig = await nearRpc.sendJsonRpc(
+    "EXPERIMENTAL_genesis_config",
+    {}
+  );
   const networkProtocolConfig = await nearRpc.sendJsonRpc(
     "EXPERIMENTAL_protocol_config",
     { finality: "final" }
   );
   const epochStatus = await nearRpc.sendJsonRpc("validators", [null]);
-  const numSeats =
+  const maxNumberOfSeats =
     networkProtocolConfig.num_block_producer_seats +
     networkProtocolConfig.avg_hidden_validator_seats_per_shard.reduce(
```

Comment:
@frol I just want to clarify that params `num_block_producer_seats` and  `avg_hidden_validator_seats_per_shard` in `near-cli` were received from [EXPERIMENTAL_genesis_config](https://github.com/near/near-cli/blob/master/utils/validators-info.js#L15) and not from `EXPERIMENTAL_protocol_config` as we have. I compared the calculation of `maxNumberOfSeats` from  `EXPERIMENTAL_genesis_config` and `EXPERIMENTAL_protocol_config` and both values are the same. But is there any difference or we can leave it as is?